### PR TITLE
more graph metrics

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -22,51 +22,74 @@
...
+    resp.headers["X-Thoth-Metrics-Exporter-Version"] = f"v{__version__}"
 
 
-@application.route('/')
-def main():
+@api.route("/")
+async def main(req, resp):
     """Show this to humans."""
-    return "This service is not for humans!"
+    resp.text = "This service is not for humans!"
```
Comment (fridex) [inline app.py]: Just an idea - what about redirecting to grafana (I think we don't have the url in deployment so its not that clear, right?)?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[tool.black]
```
Comment (fridex) [inline pyproject.toml]: :+1:

File: thoth/metrics_exporter/jobs.py [hunk 3]
```diff
@@ -64,55 +69,89 @@ def countJobStatus(JobListItems: dict) -> (int, int, int):
...
-        _LOGGER.error(excpt)
 
 def get_janusgraph_v_and_e_total():
     """Get the total number of Vertices and Edges stored in JanusGraph Server."""
-    graph_db = GraphDatabase.create('test.janusgraph.thoth-station.ninja', port=8182)
-    graph_db.connect() # FIXME no hardcoded hostnames
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+
+    graph_db = GraphDatabase.create("janusgraph", port=8182)
```
Comment (fridex) [inline thoth/metrics_exporter/jobs.py]: a TODO for not hardcoded name?

Comment (goern) [reply thoth/metrics_exporter/jobs.py]: well it's the service name and service port, this is provided/injected by OpenShift and is always present... all other services depend on it too

Comment (fridex) [reply thoth/metrics_exporter/jobs.py]: So pick it from env variables (that's what I was referring to)?

Comment (goern) [reply thoth/metrics_exporter/jobs.py]: ah, right... you refer to https://github.com/thoth-station/storages/blob/master/thoth/storages/graph/janusgraph.py#L94

File: app.py (hunk 1 above)
Comment (CermakM) [reply app.py]: +1 On the grafana, I coincidentally struggled to find out the grafana url few moments ago.

Comment (goern) [reply app.py]: done with https://github.com/thoth-station/metrics-exporter/pull/6/commits/7060984de178ff4a9c98183f8ad89df49c56c581

**Comment by fridex** [comment]
@goern I've rebased this one against the current master. Please have a look at changes.

**Comment by goern** [comment]
regate

**Pull request merged**